README.md

```diff
@@ -213,7 +213,11 @@ Here is the general syntax for a JSON input schema:
   {
     "_meta": {
       "topic": "<my kafka topic>",
-      "key": "<field to be used for kafka record key>" ,
+      "key": "<field to be used for kafka record key>" , 
+      "proto": {
```

Comment:
It’s ok to add an example that uses protobuf output format, but we shouldn’t make breaking changes to what is required for the input json schema.

Reply:
I am updating the README to add a separate section called:
### JSON Schema with Protobuf.

The `proto {}` meta field is intended to be optional.